Dockerfile-pytorch-ngc

```diff
@@ -20,7 +20,8 @@ RUN python -m pip install determined && python -m pip uninstall -y determined
 RUN python -m pip install  \
     -r /tmp/det_dockerfile_scripts/additional-requirements-torch.txt \
     -r /tmp/det_dockerfile_scripts/additional-requirements.txt \
-    -r /tmp/det_dockerfile_scripts/notebook-requirements.txt
+    -r /tmp/det_dockerfile_scripts/notebook-requirements.txt && \
+    jupyter labextension disable "@jupyterlab/apputils-extension:announcements"
```

Comment:
Can we make this a separate `RUN` line and maybe add a comment? I want these files to be quickly understandable for external users.

Reply:
Applies to `tensorflow-ngc` as well.